### ISSUE TITLE
bpgl_algo ImportError

### ISSUE THREAD

**Issue by NoahRJohnson** [VisionSystemsInc/pyvxl#119]
`(src) user@23b7bdc9dd09:/src$ python3
Python 3.8.8 (default, Feb 24 2021, 21:46:12) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import vxl.contrib.bpgl.algo as bpgl_algo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/install/lib/python3/site-packages/vxl/contrib/bpgl/algo/__init__.py", line 1, in <module>
    from ._bpgl_algo import *
ImportError: arg(): could not convert default argument into a Python object (type not registered yet?). Compile in debug mode for more information.`

**Comment by drewgilliam** [comment]
Compiling in debug mode gives a more specific traceback
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/install/lib/python3/site-packages/vxl/contrib/bpgl/algo/__init__.py", line 2, in <module>
    from ._bpgl_algo import *
ImportError: arg(): could not convert default argument 'target_window: vgl_box_2d<int>' in method '<class 'vxl.contrib.bpgl.algo._bpgl_algo.rectify_image_pair_affine'>.process' into a Python object (type not registered yet?)
```

**Comment by drewgilliam** [comment]
fixed with PR #122
